CryptoPkg/Library/BaseCryptLibMbedTls/Pk/CryptPkcs7VerifyCommon.c

```diff
@@ -555,7 +557,11 @@ MbedtlsPkcs7SignedDataVerifySigners (
   ZeroMem(Hash, MBEDTLS_MD_MAX_SIZE);
   mbedtls_md (MdInfo, Data, DataLen, Hash);
   if (SignerInfo->AuthAttr.p != NULL) {
+    TempAuthAttr = *(SignerInfo->AuthAttr.p);
+    *(SignerInfo->AuthAttr.p) = 0x31;

```

Comment:
please use MACRO.

Reply:
Thanks.  I have submitted the PR:  https://github.com/tianocore/edk2-staging/pull/459